src/couch/src/couch_changes.erl

```diff
@@ -368,6 +395,17 @@ open_revs(Db, DocInfo, Style) ->
     OpenResults = [couch_db:open_doc(Db, DI, OpenOpts) || DI <- DocInfos],
     [Doc || {ok, Doc} <- OpenResults].
 
+open_revs2(Db, DocInfo) ->
+    DocInfos = [DocInfo#doc_info{revs = [R]} || R <- DocInfo#doc_info.revs],
+    OpenOpts = [deleted, conflicts],
+    OpenResults = [couch_db:open_doc(Db, DI, OpenOpts) || DI <- DocInfos],
+    Docs = [Doc || {ok, Doc} <- OpenResults],
+    [Doc1 | RestDocs] = Docs,
+    RestRevs = [Doc#doc.revs || Doc <- RestDocs],
+    Conflicts = [{conflicts, [{Pos, RevId} || {Pos, [RevId]} <- RestRevs]}],
```

Comment:
Wonder if this will add in deleted conflicts as well. Not sure if we have a test case to check for live vs deleted conflicts but I think we should ensure we follow the exact previous behavior if we emitted deleted conflicts, I think this would work but if we didn't previously we'd be breaking the API

Reply:
It seems the `deleted_conflicts` parameter only exists in [`GET /{db}/{docid}`](https://docs.couchdb.org/en/latest/api/document/common.html#get--db-docid) endpoint.
We can't use it in `_changes`.
Added a filter in `RestRevs` to remove deleted conflict revs from the `_conflicts` field.
Thank you for the review!

Reply:
That works! Nicely done.